src/test/java/teammates/test/cases/datatransfer/StudentProfileAttributesTest.java

```diff
@@ -295,6 +295,35 @@ public void testUpdateOptionsBuilder_withNullInput_shouldFailWithAssertionError(
                         .withPictureKey(null));
     }
 
+    @Test
+    public void testEquals() {
```

Comment:
You can test: `hashCode()` of objects should be equal if there are equal.

Reply:
Added tests for `hashCode()`